pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/ProtoParserTools.java

```diff
@@ -200,7 +200,27 @@ public static double readDouble(final ReadableSequentialData input) {
      * @return Read string
      */
     public static String readString(final ReadableSequentialData input) throws IOException {
+        try {
+            return readString(input, Long.MAX_VALUE);
```

Comment:
Don't love the calling here with Long.MAX_VALUE to do the equivalent of disabling check as we still pay performance cost.

Reply:
Yes, I was thinking the same when making this change. I decided to go with reusing the code instead of making a copy for a simple reason: in an ideal world, nobody should be calling this particular method outside of PBJ, and PBJ Codecs now only call the maxSize-version of this method now.

So as far as parsing with PBJ codecs is concerned, the performance should be top-notch. And those unlucky who decided to call ProtoParserTools methods directly from elsewhere should reconsider their choices. :)